crane_x7_tests/setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import find_packages, setup
+
+package_name = 'crane_x7_tests'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='decarabas',
+    maintainer_email='codrea.eric@gmail.com',
+    description='TODO: Package description',
+    license='TODO: License declaration',
```

Comment:
what is the license used in the rest of the Crane package?

Reply:
Apache license 2.0. I added the same one to ours so we are congruent with what they do